migrate-abs-to-spe.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Sample App To Migrate from Azure Blob Storage container to SharePoint Embedded container 
+description: Tutorial in how to migrate from Azure Blob Storage container to SharePoint Embedded container
+ms.date: 07/31/2024
+ms.localizationpriority: high
+---
+
+# Introduction
+
+## Purpose
+
+This tutorial will guide you through the process of migrating content from Azure Blob Storage (ABS) to SharePoint Embedded (SPE). This is particularly useful for smaller customers who need to move their content efficiently.
+
+## Prerequisites
+
+1. .NET Core SDK
+1. Dotnet environment to run sample app
+    - It can be run Windows, Linux and MacOs
+1. SharePoint Embedded container
+    - For more information how to set up [SPE container](http://aka.ms/start-spe)
+1. Azure Blob Storage container
+    - For more information how to set up [ABS container](https://learn.microsoft.com/azure/storage/blobs/storage-blobs-introduction)
+
+# Authenication
+
+## Azure Blob Storage
+
+1. Credentials - Container level Shared Access Signature (SAS) URL.
+1. Permission - Read and List
+
+## SharePoint Embedded site
+
+1. App type - A guest app
+1. Credentials - SPE client id and SPE container
+    -  Have permissions to Write and Create folder in the container
+1. Permission - "User.Read", "FileStorage.Container.Selected", "Files.ReadWrite"
+
+# Overview of the Sample App
+
+## Description
+
+The sample app is called "MigrateABStoSPE" and it is designed to migrate files from an Azure Blob Storage (ABS) container to a SharePoint Online (SPE) container. It uses Azure.Storage.Blobs and Newtonsoft.Json libraries for working with ABS and JSON data respectively. The app authenticates with both ABS and SPE using client credentials and performs the migration of files.
+
+## Running the Sample App
+
+1.	Open a terminal or command prompt.
+1.	Navigate to the directory where the Program.cs file is located.
+1.	Make sure you have the .NET Core SDK installed on your machine. You can check this by running the command dotnet --version in the terminal. If the command is not recognized, you can download and install the .NET Core SDK from the official Microsoft website.
+1.	Once you have confirmed that the .NET Core SDK is installed, you can build the application by running the command `dotnet build`. This will compile the code and generate the necessary binaries.
```

Comment:
Do you need to build it ahead of time or can you do dotnet run? I think you don’t need to do dotnet build if you do dotnet run. Instead, you may just need to install dependencies. But be clear because you are compiling the binary and then ignore the binary and run Program.cs

also, add run instructions at the end once you have guided the user through the tutorial. We want them to learn, that’s the main goal. Being able to run the app conveniently is a good bonus.